vignettes/releases/vdiffr-0.1.0.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "vdiffr 0.1.0"
+---
+
+We are very pleased to announce the first release of vdiffr, an
+extension to the testthat package that makes it easy to add visual
+unit tests for plots.
```

Comment:
Need a sentence or two on why this is important/hard. "Testing visualisations is hard because ..."